rust/src/delta.rs

```diff
@@ -547,7 +547,7 @@ impl DeltaTable {
 
     async fn get_latest_version(&mut self) -> Result<DeltaDataTypeVersion, DeltaTableError> {
         let mut version = match self.get_last_checkpoint().await {
-            Ok(last_check_point) => last_check_point.version,
+            Ok(last_check_point) => last_check_point.version + 1,
```

Comment:
Not sure if I misunderstand it. But by comparing with other related methods, this looks incorrect to me.

Reply:
I think this is a valid optimization. It should save us one unnecessary object head call because for a valid delta table, the version stored in the checkpoint should always point to a valid commit object.